install.sh

```diff
@@ -8,7 +8,8 @@ case $(uname -sm) in
 	"Darwin x86_64") target="darwin-amd64" ;;
 	"Darwin arm64") target="darwin-arm64" ;;
 	"Linux x86_64") target="linux-amd64" ;;
-	*) echo "Unsupported platform $(uname -sm). Only Darwin x86_64, Darwin arm64 and Linux x86_64 binaries are available."; exit ;;
+	"Linux arm64") target="linux-arm64" ;;
```

Comment:
`Linux aarch64` since you didn't know how it's called

Reply:
oh, thanks